admin_manual/configuration/server/antivirus_configuration.rst

```diff
@@ -30,23 +30,33 @@ ownCloud integrates with anti-virus tools by connecting to them via:
 
 In the case of ClamAV, ownCloud's Antivirus extension sends files as streams to a ClamAV service (which can be on the same ownCloud server or another server within the same network) which in turn scans them and returns a result to stdout. 
 
-The information is then parsed or an exit code is evaluated if no result is available to determine the response from the scan. 
-Based on ownCloud’s evaluation of the response (or exit code) an appropriate action is then taken, such as recording a log message or deleting the file. 
+.. note:: 
+   Individual chunks are **not** scanned. The whole file is scanned when it is moved to the final location.
 
-.. important::
-   Files are checked when they are uploaded or updated (whether because they were edited or saved) but *not* when they are downloaded. And ownCloud doesn't support a file cache of previously scanned files.
+The information is then parsed or an exit code is evaluated if no result is available to determine the response from the scan. 
+Based on ownCloud's evaluation of the response (or exit code) an appropriate action is then taken, such as recording a log message or deleting the file. 
 
 .. note::
    Scanner exit status rules are used to handle errors when ClamAV is run in CLI mode. 
    Scanner output rules are used in daemon/socket mode.
 
+Things To Note
+~~~~~~~~~~~~~~
+
+#. Files are checked when they are uploaded or updated (whether because they were edited or saved) but *not* when they are downloaded. 
+#. ownCloud doesn't support a cache of previously scanned files.
+#. If the app is either not configured or is misconfigured, then it rejects file uploads.
+#. If ClamAV is unavailable, then it rejects file uploads.
```

Comment:
"If ClamAV is unavailable, then it rejects " it -> the app

Reply:
gotcha.